packages/core/src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -479,16 +481,35 @@ export async function startTransformationJob(uploadId: string) {
         getLogger().info(`CodeTransformation: jobId: ${jobId}`)
     } catch (error) {
         getLogger().error(`CodeTransformation: ${CodeWhispererConstants.failedToStartJobNotification}`, error)
-        if ((error as Error).message.includes('too many active running jobs')) {
+        const errorMessage = (error as Error).message
+        if (errorMessage.includes('too many active running jobs')) {
             transformByQState.setJobFailureErrorNotification(
                 CodeWhispererConstants.failedToStartJobTooManyJobsNotification
             )
             transformByQState.setJobFailureErrorChatMessage(
                 CodeWhispererConstants.failedToStartJobTooManyJobsChatMessage
             )
+        } else if (errorMessage.includes('Monthly aggregated Lines of Code limit breached')) {
+            transformByQState.setJobFailureErrorNotification(
+                CodeWhispererConstants.failedToStartJobMonthlyLimitNotification
+            )
+            transformByQState.setJobFailureErrorChatMessage(
+                CodeWhispererConstants.failedToStartJobMonthlyLimitChatMessage
+            )
```

Comment:
Sorry didn't catch this last time, but Lines of Code limit is checked during the transformation build phase and will be returned as a transformation failure reason, instead of a validation error from start job. These should be handled during transformation polling.